bugzilla.js

```diff
@@ -55,13 +70,46 @@ function addChoice(form, value, text) {
       alert('You need to select something!');
     }
 
-    browser.storage.sync.set({
-      [bugId]: categorization,
+    browser.storage.sync.get("categorization").then(function(data){
```

Comment:
Instead of `categorization`, which is too generic, call this `regression_bug_nobug`